.github/workflows/deploy-staging.yaml

```diff
@@ -204,6 +204,21 @@ jobs:
           enforce_admins: true
           retries: 8
 
+      - id: install-fluxctl
+        name: Install fluxctl
+        run: |-
+          sudo snap install fluxctl --classic
+
+      - id: add-kubeconfig
+        name: Add k8s config file for existing cluster.
+        run: |-
+          aws eks update-kubeconfig --name biomage-staging
+
+      - id: update-flux-namespace
+        name: Update flux staging
+        run: |-
+          fluxctl sync --k8s-fwd-ns flux --context arn:aws:eks:eu-west-1:242905224710:cluster/biomage-staging
```

Comment:
Let's fetch the account ID dynamically here. I can see it's also used in another part of this pipeline.

Doing something like:

```
AWS_ACCOUNT_ID=$(aws sts get-caller-identity | cut -f1)
```

will give you an env variable with the account corresponding to the AWS credentials you are authenticated with.